MAINTAINER.md

```diff
@@ -0,0 +1,15 @@
+# Instructions for maintainers of the nodereport project
+
+## Releasing to www.npmjs.com

```

Comment:
I just took the URL from my browser, npmjs.org re-directs immediately to npmjs.com. 

OK, I will change to 'publish to npm', people know what that means.